.github/workflows/regression_tests.yml

```diff
@@ -33,6 +33,17 @@ jobs:
           path: cloudsc
           ref: develop
 
+      - name: Clone ECWAM
+        run: |
+          mkdir ${{ github.workspace }}/ecwam
+          mkdir -p ~/.ssh/ && touch ~/.ssh/known_hosts
+          cat << 'EOF' > ~/.ssh/known_hosts
+          ${{ secrets.BITBUCKET_HOSTKEY }}
+          EOF
+          eval `ssh-agent -s`
```

Comment:
Is there a specific reason why this is wrapped in an `eval`?

Reply:
The `eval` is to load the environment variables output by `ssh-agent`.

Reply:
Ah, of course!